schemas/com.snowplowanalytics.snowplow.enrichments/pii_enrichment_config/jsonschema/2-0-1

```diff
@@ -0,0 +1,168 @@
+                "description": "JSON field which contains a JSON string value, on which pseudonymization will be performed on a specific JSON path",
+                "type": "object",
+                "properties": {
+                  "field": {
+                    "enum": [
+                      "contexts",
+                      "derived_contexts",
+                      "unstruct_event"
+                    ]
+                  },
+                  "schemaCriterion": {
+                    "type": "string",
+                    "pattern": "^iglu:([a-zA-Z0-9-_.]+)/([a-zA-Z0-9-_]+)/([a-zA-Z0-9-_]+)/([1-9][0-9]*|\\*)-((?:0|[1-9][0-9]*)|\\*)-((?:0|[1-9][0-9]*)|\\*)$"
+                  },
+                  "jsonPath": {
+                    "type": "string",
+                    "pattern": "^\\$.*$"
+                  }
+                },
+                "required": [
+                  "field",
+                  "schemaCriterion",
+                  "jsonPath"
+                ],
+                "additionalProperties": false
+              }
+            },
+            "oneOf": [
+              {
+                "required": [
+                  "pojo"
+                ]
+              },
+              {
+                "required": [
+                  "json"
+                ]
+              }
+            ],
+            "additionalProperties": false
+          }
+        },
+        "strategy": {
+          "description": "The pseudonymization strategy which will be applied to all the fields specified in the `pii` section",
+          "type": "object",
+          "properties": {
+            "pseudonymize": {
+              "description": "Pseudonymization strategy that hashes using a specified algorithm",
+              "type": "object",
+              "properties": {
+                "hashFunction": {
+                  "description": "The hash function that will be used by this strategy",
+                  "enum": [
+                    "MD2",
+                    "MD5",
+                    "SHA-1",
+                    "SHA-256",
+                    "SHA-384",
+                    "SHA-512"
+                  ]
+                },
+                "salt": {
+                  "description": "A salt that will be added to the field durring hashing",
+                  "type": "string"
+                }
+              },
+              "required": [
+                "hashFunction",
+                "salt"
+              ],
+              "additionalProperties": false
+            }
+          },
+          "required": [
+            "pseudonymize"
+          ],
+          "additionalProperties": false
+        }
+      },
+      "required": [
+        "pii",
+        "strategy"
+      ],
+      "additionalProperties": false
+    },
+    "anonymousOnly": {
+      "type": "boolean",
+      "description": "Whether to enable anonymous mode. If enabled, the enrichment masks PII when the SP-Anonymous header is present"
```

Comment:
```suggestion
      "description": "Whether to enable anonymous mode. If enabled, the enrichment masks PII only when the SP-Anonymous header is present"
```

Reply:
thanks @benjben ! I updated description and force-pushed.